lib/src/widgets/components/request_item_card/request_item_card_owner.dart

```diff
@@ -0,0 +1,21 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_request_kit/flutter_request_kit.dart';
+import 'package:flutter_request_kit/src/extension/provider_extensions.dart';
+
+class RequestItemCardOwner extends StatelessWidget {
+  const RequestItemCardOwner({super.key, required this.userName});
+
+  final String userName;
+
+  @override
+  Widget build(BuildContext context) {
+    final theme = context.theme.extension<RequestItemCardTheme>();
+
+    return Text(
+      'by $userName',
```

Comment:
This needs translations tho we can omit this for now. if you can add a TODO that would be nice.